plugins/inputs/filestat/filestat_test.go

```diff
@@ -25,17 +26,19 @@ func TestGatherNoMd5(t *testing.T) {
 		"file": dir + "log1.log",
 	}
 	fields1 := map[string]interface{}{
-		"size_bytes": int64(0),
-		"exists":     int64(1),
+		"size_bytes":        int64(0),
+		"exists":            int64(1),
+		"modification_time": int64(getModificationTime(fs, tags1["file"])),
```

Comment:
Just use `acc.HasInt64Field(` to assert that the field is set, then you can remove the getModificationTime function.  This will be good enough for now in my opinion.

Reply:
I was hoping you wouldn't ask for something like that 😉 

means I need to modify each test case to assert all fields individually, because `acc.AssertContainsTaggedFields(...)` will fail now.  Hence why I did the getModificationTime function... required fewer changes to implement.